components/Table/Table.js

```diff
@@ -271,6 +274,33 @@ export class Table {
     }
   }
 
+  renderModalWindow(matchObj, rowId) {
```

Comment:
You have the class `Modal` and this should be his responsibility to create his basic DOM